src/base/rss/rssdownloadrule.cpp

```diff
@@ -42,6 +42,35 @@
 
 using namespace Rss;
 
+const QString episodeRegex = "[^A-Za-z0-9]((?:S\\d{1,}(?:E\\d{1,})+)|(?:\\d{1,}x\\d{1,})|(?:\\d{4}\\.\\d{2}\\.\\d{2}))[^A-Za-z0-9]";
+
+QString computeEpisodeName(QString matchedEpisode)
```

Comment:
I don't think changing this to a reference makes any real difference, as either way its initializing one `QString` using the given c string. However I could see a point to making it `static const QString` just to make sure it doesn't accidentally leak into any other file.